core/flow/rule_manager_test.go

```diff
@@ -169,9 +171,7 @@ func TestGetRules(t *testing.T) {
 		assert.True(t, len(tcMap["abc2"]) == 1 && !tcMap["abc2"][0].boundStat.reuseResourceStat)
 		assert.True(t, reflect.DeepEqual(tcMap["abc2"][0].boundStat.readOnlyMetric, nopStat.readOnlyMetric))
 		assert.True(t, reflect.DeepEqual(tcMap["abc2"][0].boundStat.writeOnlyMetric, nopStat.writeOnlyMetric))
-		if err := ClearRules(); err != nil {
```

Comment:
If we remove ClearRules() there,   Maybe we should add ClearRules() test case,

Reply:
`clearData()` do it~